test/acceptance/error-handling.acceptance.ts

```diff
@@ -0,0 +1,31 @@
+import {expect} from '@loopback/testlab';
+import * as cp from 'child_process';
+import * as path from 'path';
+import * as http from 'http';
+
+describe('App error handler (acceptance)', () => {
+  let child: cp.SpawnSyncReturns<Buffer>;
```

Comment:
FWIW, since the `child` is used inside test only, I think there is no need to define it at `describe` suite level.

```ts
// inside the test
const child = cp.spawnSync(//...
```